### PR TITLE
Added readme 

### PR DESCRIPTION
**Pull request by AlexYanski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+
+Bootstrap project is a 
+
+# How to use
+
+## Setup
+Clone this repository
+
+```bash
+git clone https://github.com/touchlane/Bootstrap-iOS.git
```
Comment (pkondrashkov) [inline README.md]: This won't install submodules

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+
+# Requirements
+
+* iOS 9.0+
+* Xcode 10.2+
+* Swift 5.0+
+
+# How it works
+
+Bootstrap project is a
```
Comment (pkondrashkov) [inline README.md]: Why is this empty?

File: README.md (hunk 1 above)
Comment (AlexYanski) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (AlexYanski) [reply README.md]: Fixed

**Pull request merged**